utility/src/test/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="org.oppia.util">
   <uses-sdk android:minSdkVersion="19"
-    android:targetSdkVersion="29" />
+    android:targetSdkVersion="28" />
```

Comment:
Note: Utility tests do not pass if targetSDK is 29.

Reply:
For context, the tests build but do not pass. This is the error: 
```
2) testFormatDateFromDateString_successFormatToString(org.oppia.util.system.OppiaDateTimeFormatterTest)
java.lang.IllegalArgumentException: API level 29 is not available
```

Reply:
Ah, that's probably because the workspace is setup to use 28: https://github.com/oppia/oppia-android/blob/develop/WORKSPACE#L13. That's fine--let's keep with 28 if it's a hard blocker right now. We can figure out how to support a wider band of versions later.

Reply:
I tried altering the WORKSPACE to use 29 to see if the error would resolve but it didn't. I will leave it at 28 for now, then :)

Reply:
@anandwana001